docs/cubeviz/export_data.rst

```diff
@@ -22,13 +22,19 @@ can be used to extract the *spectrum* of a spatial subset named "Subset 1":
 
 .. code-block:: python
 
-    subset1_spec1d = cubeviz.specviz.get_spectra("Subset 1")
+    subset1_spec1d = cubeviz.specviz.get_spectra(subset_to_apply="Subset 1")
 
 An example without accessing Specviz:
 
 .. code-block:: python
 
-    subset1_spec1d = cubeviz.app.get_data_from_viewer("flux-viewer", data_label="Subset 1")
+    subset1_spec1d = cubeviz.get_data(data_label=flux_data_label, 
+                                      subset_to_apply="Subset 1",
+                                      statistic="mean")
+
+Note that in the above example, the ``statistic`` keyword is used to tell Cubeviz
+how to collapse the flux cube down to a one dimensional spectrum - this does not
+automatically match the collapse function used in the spectrum viewer.
```

Comment:
I wasn't sure exactly what you meant here, so I'm tweaking it with my interpretation?
```suggestion
how to collapse the flux cube down to a one dimensional spectrum - this is not 
necessarily equivalent to the collapsed spectrum in the spectrum viewer, which 
may have used a different collapse statistic.
```